sw/device/lib/base/mmio.h

```diff
@@ -234,6 +234,62 @@ inline void mmio_region_nonatomic_set_mask32(mmio_region_t base,
   mmio_region_write32(base, offset, value);
 }
 
+/**
+ * Masked Register field.
+ */
+typedef struct mmio_region_field32 {
+  uint32_t mask;  /**< The field mask. */
+  uint32_t index; /**< The field position within the a register. */
+  uint32_t value; /**< The field value to be written. */
+} mmio_region_field32_t;
+
+/**
+ * Sets the |field| from the MMIO region |base| at the given offset.
```

Comment:
Please switch to backticks or `@p` for monospace - make this change to all comments you add in this PR (existing comments shouldn't be updated). Also remember to monospace `offset`.

Reply:
I can do that, my thinking was to keep it the same as the other descriptions and change them all in one go later.

Reply:
> Also remember to monospace offset

What do you mean? Done the rest.